source/backend/api/Services/DocumentFileService.cs

```diff
@@ -74,376 +77,143 @@ public IList<T> GetFileDocuments<T>(FileType fileType, long fileId)
                 case FileType.Acquisition:
                     User.ThrowIfNotAuthorized(Permissions.AcquisitionFileView);
-                    return _acquisitionFileDocumentRepository.GetAllByAcquisitionFile(fileId).Select(f => f as T).ToArray();
+                    return _acquisitionFileDocumentRepository.GetAllByParentId(fileId).Select(f => f as T).ToArray();
                 case FileType.Project:
                     User.ThrowIfNotAuthorized(Permissions.ProjectView);
-                    return _projectRepository.GetAllProjectDocuments(fileId).Select(f => f as T).ToArray();
+                    return _projectDocumentRepository.GetAllByParentId(fileId).Select(f => f as T).ToArray();
                 case FileType.Lease:
                     User.ThrowIfNotAuthorized(Permissions.LeaseView);
-                    return _leaseRepository.GetAllLeaseDocuments(fileId).Select(f => f as T).ToArray();
+                    return _leaseFileDocumentRepository.GetAllByParentId(fileId).Select(f => f as T).ToArray();
                 case FileType.ManagementActivity:
                     User.ThrowIfNotAuthorized(Permissions.ManagementView);
-                    return _propertyActivityDocumentRepository.GetAllByPropertyActivity(fileId).Select(f => f as T).ToArray();
+                    return _propertyActivityDocumentRepository.GetAllByParentId(fileId).Select(f => f as T).ToArray();
                 case FileType.ManagementFile:
                     User.ThrowIfNotAuthorized(Permissions.ManagementView);
-                    return _managementFileDocumentRepository.GetAllByManagementFile(fileId).Select(f => f as T).ToArray();
+                    return _managementFileDocumentRepository.GetAllByParentId(fileId).Select(f => f as T).ToArray();
                 case FileType.Disposition:
                     User.ThrowIfNotAuthorized(Permissions.DispositionView);
-                    return _dispositionFileDocumentRepository.GetAllByDispositionFile(fileId).Select(f => f as T).ToArray();
+                    return _dispositionFileDocumentRepository.GetAllByParentId(fileId).Select(f => f as T).ToArray();
+                case FileType.Property:
+                    User.ThrowIfNotAuthorized(Permissions.PropertyView);
+                    return _propertyDocumentRepository.GetAllByParentId(fileId).Select(f => f as T).ToArray();
                 default:
-                    throw new BadRequestException("FileT type not valid to get documents.");
+                    throw new BadRequestException("File type not valid to get documents.");
             }
         }
 
         public async Task UploadAcquisitionDocument(long acquisitionFileId, DocumentUploadRequest uploadRequest)
         {
             Logger.LogInformation("Uploading document for single acquisition file");
             User.ThrowIfNotAllAuthorized(Permissions.DocumentAdd, Permissions.AcquisitionFileEdit);
-            ValidateDocumentUpload(uploadRequest);
```

Comment:
All of the calls were doing the same thing. The new generic method handles it and these ones maintain logs and permisions